tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "React Library",
```

Comment:
Should we rename this to SigNoz Design Tokens ?

Reply:
Very minor comment, can be taken up in next PR.

Reply:
Done, @YounixM.